examples/mock/mockoon-digitalregistries.json

```diff
@@ -14,8 +14,8 @@
       "endpoint": "data/MyPersonalDataUsage/1.0",
       "responses": [
         {
-          "uuid": "af65df73-2417-459e-8156-725979689d70",
-          "body": "[\n  {\n    \"ID\": \"1234567\",\n    \"ReaderID\": \"{{queryParam 'userID'}}\",\n    \"ReaderInitials\": \"JD\",\n    \"ReaderInstitutionID\": \"EE70049837\",\n    \"ReaderInstitutionName\": \"East Hospital\",\n    \"ReaderApplicationName\": \"East Hospital healthcare back office\",\n    \"SearchDateTime\": \"2017-07-21T17:32:28Z\",\n    \"Refrences\": [\n      {\n        \"ReferenceID\": \"{{queryParam 'DatabaseID'}}\"\n      }\n    ]\n  }\n]",
+          "uuid": "ad5ddc6e-bd4f-491b-b600-5cab337701f5",
+          "body": "[\n  {\n    \"ID\": \"1234567\",\n    \"ReaderID\": \"{{queryParam 'userID'}}\",\n    \"ReaderInitials\": \"JD\",\n    \"ReaderInstitutionID\": \"EE70049837\",\n    \"ReaderInstitutionName\": \"East Hospital\",\n    \"ReaderApplicationName\": \"East Hospital healthcare back office\",\n    \"SearchDateTime\": \"2017-07-21T17:32:28Z\",\n    \"Refrences\": [\n      {\n        \"ReferenceID\": \"MCTS311\"\n      }\n    ]\n  }\n]",
```

Comment:
I think 
```
ReaderID
```
won't be the same as this one from the request. 

Request -> User's personal unique identifier (User that wants to see who has read his personal data)
Response -> Identifier of a reader who looked at personal data